src/lib/instructors.ts

```diff
@@ -25,6 +25,8 @@ export type Instructor = {
   avatar_64_url: string
 }
 
+// HIDE_DUPLICATES: there doesn't appear to be anywhere in the app that imports
```

Comment:
Based on the code, I'm assuming that if we did use this somewhere, it *would* need to be de-duped?

Reply:
I think that this was removed in favor of pointing at search page